src/base/utils/fs.cpp

```diff
@@ -518,3 +519,9 @@ QString Utils::Fs::cacheLocation()
         locationDir.mkpath(locationDir.absolutePath());
     return location;
 }
+
+QString Utils::Fs::configurationName()
+{
+    QString envConfName = QString::fromLocal8Bit(qgetenv("QBT_CONF_NAME"));
```

Comment:
Change this in runtime will cause undefined behaviour or even crash the program. So it must be initialized at program startup and do not change further.